# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
# Commit: 9d637c05af7f
# Full Hash: 9d637c05af7f8c39af78b290335ca55994849ee8
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r ea88b243717a -r 9d637c05af7f gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp
--- a/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Fri Nov 06 00:00:47 2020 +0000
+++ b/gfx/webrender_bindings/RenderCompositorD3D11SWGL.cpp	Fri Nov 06 00:01:36 2020 +0000
@@ -44,6 +44,7 @@
       mCompositor(aCompositor),
       mContext(aContext) {
   MOZ_ASSERT(mContext);
+  mSyncObject = mCompositor->GetSyncObject();
 }
 
 RenderCompositorD3D11SWGL::~RenderCompositorD3D11SWGL() {